document/core/exec/instructions.rst

```diff
@@ -3052,18 +3052,18 @@ Throwing an exception with :ref:`tag address <syntax-tagaddr>` :math:`a`
 
 .. _exec-caughtadm:
 
-Exiting a |CAUGHTadm|
-.....................
+Exiting a catch clause |CAUGHTadm|
```

Comment:
Let's remove the `|CAUGHTadm|` here, since we do not use similar keywords elsewhere

Reply:
Done.